bin/reth/src/args/utils.rs

```diff
@@ -44,8 +44,11 @@ pub fn genesis_value_parser(s: &str) -> eyre::Result<Arc<ChainSpec>, eyre::Error
         "holesky" => HOLESKY.clone(),
         "dev" => DEV.clone(),
         _ => {
-            let raw = fs::read_to_string(PathBuf::from(shellexpand::full(s)?.into_owned()))?;
-            let genesis: AllGenesisFormats = serde_json::from_str(&raw)?;
+            let genesis: AllGenesisFormats =
+                match fs::read_to_string(PathBuf::from(shellexpand::full(s)?.into_owned())) {
```

Comment:
I'd like to see a note here about what this does.

Reply:
Added a comment